ReplayKitExample/BroadcastExtension/ExampleReplayKitAudioCapturer.m

```diff
@@ -129,9 +135,21 @@ OSStatus ExampleCoreAudioDeviceRecordCallback(CMSampleBufferRef sampleBuffer) {
     int8_t *audioBuffer = (int8_t *)bufferList.mBuffers[0].mData;
     UInt32 audioBufferSizeInBytes = bufferList.mBuffers[0].mDataByteSize;
 
+    CMFormatDescriptionRef formatDescription = CMSampleBufferGetFormatDescription(sampleBuffer);
+    const AudioStreamBasicDescription *asbd = CMAudioFormatDescriptionGetStreamBasicDescription(formatDescription);
+
+    if (asbd->mFormatFlags & kAudioFormatFlagIsBigEndian) {
+        for (int i=0; i<(audioBufferSizeInBytes-1); i += 2) {
```

Comment:
Should we have spacing between the operators? i.e. `int i=0;` => `int i = 0;`

Reply:
Maybe this is a dumb question, but should we be using an AudioConverter to swap endianness and operating out of place? i.e. Is ReplayKit expecting us to mutate the buffers that they are passing in.

Reply:
I haven't used AudioConverter before, but it sounds promising. I will see if I can use it. I guess in memory conversion is not a requirement.

Reply:
I guess one benefit is that we can think of format conversion a little more generically if we use an AudioConverter and we would be insulated if we need to change more than just endianness in the future. The code to perform endianness conversion is also probably faster than the scalar implementation proposed here. You can see examples of format conversion in AudioSinkExample.

All this being said, I'm pretty sure an AudioConverter will only operate out of place so you would need to copy the result to a temporary buffer before delivering the samples via `TVIAudioDeviceWriteCaptureData`.